binary/typedb.bat

```diff
@@ -75,6 +76,20 @@ if exist .\server\ (
   goto exiterror
 )
 
+:startcluster
+
+set "G_CP=%TYPEDB_HOME%\server\conf\;%TYPEDB_HOME%\server\lib\common\*;%TYPEDB_HOME%\server\lib\prod\*"
+
+if exist .\server\ (
```

Comment:
We can't check for the exact filename as we do in unix script (`typedb`) due to limitations of CMD

Reply:
Did you mean `if exist .\cluster\ (`?

Reply:
@lolski Nope, that's correct here: the assembled distribution of TypeDB Cluster has `server` folder too

Reply:
Ah I see.

Therefore if the user runs `typedb cluster` on a TypeDB distribution, the script will incorrectly allow it right? Unlike in the `typedb` Bash script where it correctly throws an error

Reply:
@lolski

>Therefore if the user runs typedb cluster on a TypeDB distribution, the script will incorrectly allow it right

Looks like it! However, checking for specific file isn't possible due to CMD limitations unfortunately